.envrc

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
```

Comment:
Please add some comments on what this is, what the purpose is (run bin/ysh etc. without changing $PATH)

how to use it, how to install, etc.


I've actually never used direnv before :)

Reply:
Okay this is done

Reply:
OK great, can you add the comments to the top of the script?  So people know how to use it


Right now it looks like it's only in the PR, and will get lost

Reply:
done